tests/reboot_cause_test.py

```diff
@@ -0,0 +1,68 @@
+import os
+import sys
+import textwrap
+
+import mock
+
```

Comment:
Remove this blank line. `mock` and `click` are both thrid-party packages and should be in the same group.